glicious/src/main/java/edu/grinnell/glicious/BrowserActivity.java

```diff
@@ -26,7 +26,7 @@ protected void onCreate(Bundle savedInstanceState) {
         mWebView.setWebViewClient(new WebViewClient());
         mWebView.getSettings().setJavaScriptEnabled(true);
         mWebView.setInitialScale(100);
-        mWebView.loadUrl("http://nutrition.grinnell.edu/NetNutrition/1/Mobile/Mobile");
+        mWebView.loadUrl("http://nutrition.grinnell.edu/NetNutrition/1");
```

Comment:
Please store this string as a `private static` constant, so it is easier to find.